<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Cerrar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>760</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Encender">
   <property name="geometry">
    <rect>
     <x>1009</x>
     <y>10</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Encender/Apagar Vehiculo</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>661</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Velocidad">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QLCDNumber" name="LCDVelocidad">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Gidugu</family>
         <italic>true</italic>
        </font>
       </property>
      </widget>
      <widget class="QSlider" name="SliderVelocidad">
       <property name="minimumSize">
        <size>
         <width>268</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="VelocidadLabel">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gidugu</family>
        <pointsize>17</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>KM/H</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>611</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Temperatura">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QLCDNumber" name="LCDTemperatura">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Gidugu</family>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TemperaturaLabel">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gidugu</family>
        <pointsize>17</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Â°C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>120</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Gasolina">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QLCDNumber" name="LCDGasolina_2">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Gidugu</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="value" stdset="0">
        <double>63.000000000000000</double>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="GasolinaLabel">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gidugu</family>
        <pointsize>17</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Litros</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Presion_Neumaticos">
    <item>
     <layout class="QVBoxLayout" name="Neumaticos34">
      <item>
       <widget class="QLCDNumber" name="lcdNeumatico_4"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNeumatico_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Neumaticos12">
      <item>
       <widget class="QLCDNumber" name="lcdNeumatico_1"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNeumatico_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="PresionLabel">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PSI</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Encendido_Apagado">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jamrul</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apagado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccional Izquierda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccional Derecha</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_LucesDelanteras">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>670</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Encender/Apagar luces delanteras</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>720</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Luces Altas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>690</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>690</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLucesEmergencia">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>50</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Encender/Apagar Luces de emergencia</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Temperatura:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Combustible:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Velocidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Presion:</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Presion:</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Presion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>630</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>630</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alerta Presion:</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>630</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLedLabel</class>
   <extends>QWidget</extends>
   <header>QLedLabel</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Cerrar</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderVelocidad</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LCDVelocidad</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
